### PR TITLE
Add help text for get credentials.

### PR DESCRIPTION
**Pull request by brendandburns**
Fixes https://github.com/Azure/azure-cli/issues/1621

### CODE REVIEW

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -503,6 +503,18 @@ def _create_kubernetes(resource_group_name, deployment_name, dns_name_prefix, na
 
 def k8s_get_credentials(name=None, resource_group_name=None, dns_prefix=None, location=None, user=None,
                         path=os.path.join(os.path.expanduser('~'), '.kube', 'config'), acs_info=None):
+    """Create a new Acs.
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: Like mentioned in the issue, should we mark `resource-group-name` as mandatory? also remove the `dns-prefix` and `location`?

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 2]
```diff
@@ -503,6 +503,18 @@ def _create_kubernetes(resource_group_name, deployment_name, dns_name_prefix, na
...
+    :param name: The name of the cluster.
+    :type name: str
+    :param resource_group_name: The name of the resource group. The name
+     is case insensitive.
+    :type resource_group_name: str
+    :param dns_prefix: Deprecated, do not use.
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: Up to you, but cli is still in preview, so you can just remove w/o going though `Deprecated` mode

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 3]
```diff
@@ -503,6 +503,18 @@ def _create_kubernetes(resource_group_name, deployment_name, dns_name_prefix, na
...
+    :type dns_prefix: str
+    :param location: Deprecated, do not use.
+    :type location: str
+    :param path: Path to a kubectl config file (optional)
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: you probably don't need `(optional)`. The convention is an argument is optional unless there is `[Required]` displayed

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 4]
```diff
@@ -503,6 +503,18 @@ def _create_kubernetes(resource_group_name, deployment_name, dns_name_prefix, na
 
 def k8s_get_credentials(name=None, resource_group_name=None, dns_prefix=None, location=None, user=None,
                         path=os.path.join(os.path.expanduser('~'), '.kube', 'config'), acs_info=None):
+    """Create a new Acs.
+    :param name: The name of the cluster.
+    :type name: str
+    :param resource_group_name: The name of the resource group. The name
+     is case insensitive.
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: You can skip the `is case insensitive`, it is well known the group name should be handled case insensitively. Also, by default, argument values should be handled as case insensitive if we can like those will be part of url, enum choices, etc

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py (hunk 2 above)
Comment (lostintangent) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: I'd vote to remove the deprecated flags as well

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py (hunk 1 above)
Comment (brendandburns) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: done.

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py (hunk 4 above)
Comment (brendandburns) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: done.

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py (hunk 2 above)
Comment (brendandburns) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: done.

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py (hunk 3 above)
Comment (brendandburns) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: done.

**Comment by brendandburns** [comment]
Comments addressed, please re-check.

Thanks!

**Comment by lostintangent** [comment]
@brendandburns Looks awesome! Quick question: I applied this change and then noticed an `--acs-info` flag show up now when running `az acs kubernetes get-credentials -h`. Is that expected? I don't recall that being there before, however, maybe it was and I just missed it :) Either way, would it make sense to ensure that doesn't appear in the help text as well?

screen shot 2017-01-17 at 12 13 48 pm

**Comment by brendandburns** [comment]
@lostintangent that should no longer be there.  It was in an old version of this PR. If you re-apply you shouldn't see it.

thanks
--brendan

**Comment by brendandburns** [comment]
@yugangw-msft this should be now ready to go (sorry for the delay)

**Pull request merged**